tasks/Dockerfile

```diff
@@ -60,7 +61,9 @@ RUN dnf -y update && \
     dnf -y builddep /tmp/cockpit.spec && \
     dnf clean all
 
-COPY cockpit-tasks install-service /usr/local/bin/
+RUN pip install pika
```

Comment:
What's wrong with installing the `python3-pika` package?

Reply:
Oh, no idea it existed, will change that of course.